### PR TITLE
Sort available coins for staking by coin age descending 

### PR DESCRIPTION
**Pull request by dantudor**
This is the implementation for sorting the available coins for staking by coin age decending.

### CODE REVIEW

File: src/wallet/wallet.cpp [hunk 1]
```diff
@@ -272,6 +272,8 @@ void CWallet::AvailableCoinsForStaking(vector<COutput>& vCoins, unsigned int nSp
                 }
         }
     }
+    
+    std::sort(vCoins.begin(), vCoins.end(), sortByCoinAgeDescending());
```
Comment (aguycalled) [inline src/wallet/wallet.cpp]: I would add here `if (GetBoolArg("-stakingsortbycoinage",false))`

Comment (mxaddict) [reply src/wallet/wallet.cpp]: I agree with this flag check

@dantudor can you add this flag check, then I'll approve PR

**Comment by mxaddict** [comment]
testedACK, waiting for requested change to approve

**Comment by mxaddict** [comment]
I added the flag check that @aguycalled  suggested `stakingsortbycoinage`

**Comment by mxaddict** [comment]
Will merge after Travis builds are done

**Comment by mxaddict** [comment]
This change looks `ok` to merge

**Pull request merged**